src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -50,30 +50,71 @@
  * </li>
  * </ul>
  * <p>
- * To prevent {@code FooCheck} violations from being reported write:
+ * To configure the default check
  * </p>
  * <pre>
- * &#64;SuppressWarnings("foo") interface I { }
- * &#64;SuppressWarnings("foo") enum E { }
- * &#64;SuppressWarnings("foo") InputSuppressWarningsFilter() { }
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;MemberName&quot;/&gt;
+ *   &lt;module name=&quot;ConstantName&quot;/&gt;
+ *   &lt;module name=&quot;ParameterNumber&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * Class Test {
+ *   &#64;SuppressWarnings({"membername"}) private int J;  // ok
+ *   &#64;SuppressWarnings({"constantname"}) private static final int m = 0;  // ok
+ *
+ *   &#64;SuppressWarnings("paramnum")
+ *   public void needsLotsOfParameters1(@SuppressWarnings("unused") int a,  // violation
```

Comment:
Remove `@SuppressWarnings("unused")`
We do not have violation without it, let's not confuse users

Reply:
Done